src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { PropTypes } from 'prop-types';
+
+const Form = ({ AddNewBook }) => {
+  const [newBook, setnewBook] = useState({ title: '', author: '' });
+
+  return (
+    <div id="form-container">
+      <h2>ADD NEW BOOK</h2>
+      <form
+        onSubmit={(e) => {
+          AddNewBook(e, newBook);
+          setnewBook({ title: '', author: '' });
+        }}
```

Comment:
- [ ] Kindly ensure your method  takes the inputs from the state and generate a unique id before adding them to an object